pages/docs/user-docs/docs-environment-metadata.md

```diff
@@ -6,36 +6,40 @@ folder: docs
 toc: false
 ---
 
-Singularity containers support environment variables and labels that can be added by user during the bootstrap process.
+Singularity containers support environment variables and labels that you can add to your container during the build process.
 
 {% include toc.html %}
 
 ## Environment
 
-If you are importing a Docker container, the environment will be imported as well. If you want to define custom environment variables in your bootstrap recipe file `Singularity` you can do that like this
+If you build a container from Singularity Hub or Docker Hub, the environment will be imported into the container at build time. You can also define custom environment variables in your Recipe file like so:
 
-```bash
-Bootstrap:docker
-From: ubuntu:latest
+```
+Bootstrap: shub
+From: singularityhub/ubuntu
 
```

Comment:
I wouldn't use the singularityhub user as an example - it can be confusing for the user, they might think that any container should be prefixed with "singularityhub" use vsoch/hello-world instead.

Reply:
we really just need a container that is guaranteed to persist and be updated over time. This is the image I do this with, so it makes sense. In the case that the github organization singularityhub goes away, that uri won't persist. I also built, manage, everything for singularity hub, so it's official yo!